docs/testrunner-toolkit.md

```diff
@@ -91,11 +91,7 @@ import TabItem from '@theme/TabItem';
   </tr>
   <tbody>
   <tr>
-    <td rowspan='2'>1.18.1</td>
-    <td><b>macOS:</b> 11.0</td>
-    <td>Chromium, Firefox, Webkit</td>
```

Comment:
Things seem to work OK for Chromium and Firefox on mac, maybe leave those in the docs and just remove webkit?